bio_files_processor.py

```diff
@@ -0,0 +1,65 @@
+from typing import TextIO
+
+
+def convert_multiline_fasta_to_oneline(
+    input_fasta: str, output_fasta: str = None
+) -> TextIO:
+    """
+    The function of bio_files_processor.py
+    It makes one long string out of several short strings
+
+    Parameters:
+    input_fasta (str): path to .fasta file with sequences
+    output_fasta (str = None): path to file with output of function.
+    Optional argument
+
+    Returns:
+    TextIO: file with long strings with sequences
+    """
+    if output_fasta is None:
+        output_fasta = "converted_" + input_fasta
+    else:
+        output_fasta = output_fasta + ".fasta"
+    with open(input_fasta, "r") as file, open(output_fasta, "w") as output_file:
+        output_seqs = {}
+        for line in file:
+            line = line.strip()
+            if line.startswith(">"):
+                name = line
+                output_seqs[name] = ""
+            elif name is not None:
+                output_seqs[name] += line
+            for key, value in output_seqs.items():
+                output_file.writelines(f"\n{key}\n{value}")
+
+
+def parse_blast_output(input_file: str, output_file: None) -> TextIO:
```

Comment:
```suggestion
def parse_blast_output(input_file: str, output_file: Optional[str]) -> TextIO:
```

-2 балла суммарно за вопросы к типам.